canonical-web-and-design/jaas-dashboard
# Change spacing between logo circle and text

## Done

Animate the Juju and JAAS logo text so that it maintains proper spacing when the sidebar is open, but provides appropriate spacing to avoid artifacts when the sidebar is closed.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- When the app loads the logo in the top left corner should have the appropriate spacing between the logo circle and text.
- Click to view a model details and the sidebar should animate closed and you should not see any text artifacts by the logo circle.
- Hover to expand the sidebar and the text should animate closer.
- Collapse the sidebar and the text should animate away.

## Details

Fixes #440 

## Screenshots
![Screen Shot 2020-04-20 at 3 53 49 PM](https://user-images.githubusercontent.com/532033/79804076-7e565280-8320-11ea-9ab6-73b04ab9b3d7.png)
![Screen Shot 2020-04-20 at 3 54 02 PM](https://user-images.githubusercontent.com/532033/79804077-7eeee900-8320-11ea-946d-4d12205c6c00.png)
